.env.example

```diff
@@ -1 +1 @@
-SKIP_PREFLIGHT_CHECK = true
\ No newline at end of file
+SKIP_PREFLIGHT_CHECK = true
```

Comment:
Not a big deal, but weird that this file is checked in - unrelated to the issue you're handling and doesn't seem to have any changes? Is this a Github thing?

Reply:
My best guess is that a space was added here, which caused GitHub to think something changed, heh.

Reply:
(Agree that we should remove it of course)

Reply:
Pushed up small commit because the .env.example file didn't have valid bash environment variable formatting.